controller/internal/routes/identity_api_model.go

```diff
@@ -176,13 +176,15 @@ func MapPatchIdentityToModel(id string, identity *rest_model.IdentityPatch, iden
 			Tags: identity.Tags,
 			Id:   id,
 		},
```

Comment:
ServiceHostingCosts and ServiceHostingPrecedences weren't getting propagated to the model object on patch